libs/counterfactuals/src/lib/CounterfactualChart.tsx

```diff
@@ -379,13 +379,14 @@ export class CounterfactualChart extends React.PureComponent<
   }
 
   private getCurrentClass(): string {
```

Comment:
minor nit: maybe getCurrentClass should be renamed to something else, like getCurrentLabel, since in the regression case classes don't apply like in the classification scenario?